### PR TITLE
Added createVoc RPC

### PR DESCRIPTION
**Pull request by Jouzo**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:

Added createVoc RPC

#### Which issue(s) does this PR fixes?:

Fixes part of #458

### CODE REVIEW

**Comment by Jouzo** [comment]
Error : 
```
Expected substring: "RpcApiError: 'Cannot create a voc while still in Initial Block Download', code: -10, method: createvoc"
    Received message:   "RpcApiError: 'Cannot create a cfp while still in Initial Block Download', code: -10, method: createvoc"
```

Should be fix on `ain` side. See https://github.com/DeFiCh/ain/pull/576#discussion_r672092005

File: packages/jellyfish-api-core/src/category/governance.ts [hunk 1]
```diff
@@ -52,6 +52,19 @@ export class Governance {
...
+   *
+   * @param {string} [title] Vote of confidence's title
+   * @param {UTXO[]} [utxos = []] Specific utxos to spend
+   * @param {string} [utxos.txid] The transaction id
+   * @param {number} [utxos.vout] The output number
+   * @return {Promise<string>} txid
+   */
+  async createVoc (title: string, utxos: UTXO[] = []): Promise<string> {
+    return await this.client.call('createvoc', [title, utxos], 'number')
+  }
```
Comment (siradji) [inline packages/jellyfish-api-core/src/category/governance.ts:67]: is `@param{title}` optional? I can see you made it optional in the jsDoc but not on line `65` . Adding a square bracket to a parameter in`jsDoc` cast it as optional.

File: website/docs/jellyfish/api/governance.md [hunk 2]
```diff
@@ -68,3 +68,18 @@ interface ProposalInfo {
...
 }
  ```
+
+## createVoc
+
+Creates a Vote of Confidence.
+
+```ts title="client.governance.createVoc()"
+interface governance {
+  createVoc (title: string, utxos: UTXO[]): Promise<string>
```
Comment (siradji) [inline website/docs/jellyfish/api/governance.md:78]: Same here.

File: packages/jellyfish-api-core/src/category/governance.ts (hunk 1 above)
Comment (Jouzo) [reply packages/jellyfish-api-core/src/category/governance.ts:67]: my bad, title is not optional

**Comment by canonbrother** [comment]
@Jouzo hmm... where is the `createvoc` from `ain` huh??

File: website/docs/jellyfish/api/governance.md (hunk 2 above)
Comment (jingyi2811) [inline website/docs/jellyfish/api/governance.md:78]: Should be this right? `createVoc (title: string, utxos: UTXO[] = []): Promise<string>`

Comment (Jouzo) [reply website/docs/jellyfish/api/governance.md:78]: yes, good catch !

File: packages/jellyfish-api-core/__tests__/category/governance/createVoc.test.ts [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should createVoc', async () => {
+    const proposalTx = await client.governance.createVoc('new vote of confidence')
+    await container.generate(1)
+
+    expect(typeof proposalTx).toStrictEqual('string')
+
```
Comment (jingyi2811) [inline packages/jellyfish-api-core/__tests__/category/governance/createVoc.test.ts:25]: You can see other files in the project. 
For txid, we check string and length together. For eg,

Screenshot 2021-07-27 at 11 25 37 PM


Hence, you can add `expect(proposalTx.length).toStrictEqual(64)`

**Review by jingyi2811 — APPROVED** [review]

**Review by canonbrother — COMMENTED** [review]
i'm good overall

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**